### PR TITLE
Review the **Get Started** section (User Guide)

### PR DESCRIPTION
**Pull request by HalynaDumych**
* Review and modify the **Get Started** section of the User guide
and update the navigation page according to the changes.

* Create an instruction on how to get access to the OTA Portal.

Relates-to: OTA-4584

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+
+1. Go to the xref:https://connect.ota.here.com[OTA Connect Portal].
+2. Click *Register for a HERE account*.
+3. Specify your first name, last name, and email address.
+4. Create a password for your HERE Account.
+5. Specify your date of birth and the country in which you live.
++
+NOTE: HERE needs to verify your date of birth to ensure that you are of the legal age to access and use HERE services.
+
+6. In *Security check*, enter the security code that you see on the picturegit push.
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]:
```suggestion
6. In *Security check*, enter the security code that you see in the picture.
```

File: ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc [hunk 2]
```diff
@@ -1,35 +1,33 @@
...
+|1) Provision devices|2) Upload a software version|3) Group devices|4) Create a software update
 
 |image::img::workflow-provision.png[image,100]
 Connect your devices to the OTA Connect server. The server stores the following information about each device:
 
-* The device identity,
+* The device identity
 * The hardware on the device
-* The software that it's currently running.
+* The software update that it is currently running
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]:
```suggestion
* The software version(s) that it is currently running
```

File: ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc [hunk 3]
```diff
@@ -1,35 +1,33 @@
...
 |===
-|1) Provision Devices|2) Group Devices|3) Upload software|4) Create update configuration
+|1) Provision devices|2) Upload a software version|3) Group devices|4) Create a software update
 
 |image::img::workflow-provision.png[image,100]
 Connect your devices to the OTA Connect server. The server stores the following information about each device:
 
-* The device identity,
+* The device identity
 * The hardware on the device
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]:
```suggestion
```

Comment (tkfu) [reply ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]: Ok, github's suggestion feature seems to be a bit buggy here and lost the other suggestions near this line. Here's what the list should be:

* The device name or identifier (often a VIN)
* The serial number and control unit type of each control unit in the device
* The software version that each control unit is currently running

Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/ota-workflow-campaign-manager.adoc]: Changed

File: ota-plus-web/docs/modules/ROOT/pages/get-access.adoc (hunk 1 above)
Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/get-access.adoc]: Corrected

**Pull request merged**